DragonFruit.Six.API/Dragon6Client.cs

```diff
@@ -76,13 +76,21 @@ public string AppId
                 if (Token?.Expired != false)
                 {
                     Token = GetToken();
-                    Authorization = $"Ubi_v1 t={Token.Token}";
+                    ApplyToken(Token);
                 }
             }
 
             return base.Perform<T>(requestData, token);
         }
 
+        /// <summary>
+        /// Method for setting a new token
```

Comment:
probably look to change this in the future but it should only be us using this so...